todo-app.css

```diff
@@ -0,0 +1,90 @@
+:root {
+  --purple-color: rgba(112, 35, 112, 1);
+  --gray-color: rgb(120, 120, 120);
+  --light-purple-color: rgba(238, 212, 238, 1);
+}
+
+.heading-1 {
+  text-align: center;
+  color: var(--purple-color);
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.todo-input {
+  outline: none;
+  padding-inline: 0.5rem;
+  padding-block: 0.3rem;
+}
+
+.todo-input:focus {
+  border: 2px solid var(--purple-color);
+  border-radius: 2px;
```

Comment:
border radius is not something you should change just on focus, let it be always there on the input